src/utils/roll-orb.ts

```diff
@@ -0,0 +1,104 @@
+import * as debug from 'debug';
+import * as jsyaml from 'js-yaml';
```

Comment:
This is a transitive dependency, not one in `package.json`, so we can't rely on it without adding it as a dependency.

Per some conversation here (https://github.com/electron/trop/pull/259), consider using the `yaml` package so we can unify our usage across repos.

Reply:
This still needs to be added to `package.json` and a lock file update.